### PR TITLE
update css guidelines

### PR DESCRIPTION
**Pull request by xi**
I finally took the time to update the CSS guidelines. Most important changes:

-   Rename "widget" to "component"
-   Drop "object"
-   Drop "adjustment"
-   Drop "default theme"
-   Make it a bit shorter.

I would like to also get feedback from @nicoladimarco

### CODE REVIEW

**Comment by joka** [comment]
looks nice

File: docs/development/coding_guides/css_guidelines.rst [hunk 1]
```diff
@@ -29,73 +29,63 @@ designers sometimes have very different angles on the same things.
...
-++++++++++++++
+Components
+++++++++++
 
-The layout defines the basic structure of a page. Typical layout objects
-are *header*, *main*, *aside*, and *footer*. Of most layout objects there
-is only a single instance on a page (e.g. only one header and footer).
+There are many synonyms for this in the different methodologies:
+Object (OOCSS), module (Smacss), block (BEM), atom/molecule/organism
+(Atomic Design), or pattern (pattern driven design).
```
Comment (der-john) [inline docs/development/coding_guides/css_guidelines.rst]: widget?

**Comment by der-john** [comment]
lgtm.

File: docs/development/coding_guides/css_guidelines.rst (hunk 1 above)
Comment (xi) [inline docs/development/coding_guides/css_guidelines.rst]: I cannot remember where this term came from and it is not used in any of the referenced systems. Feel free to add it to the list if you feel it is relevant.

**Comment by der-john** [comment]
time to say merge...

**Pull request merged**